modules/game/client/config/game.client.config.js

```diff
@@ -41,7 +41,3 @@ angular.module('game').factory('GameLogic', function () {
 angular.module('game').factory('Utils', function () {
   return window.utils;
 });
-
-angular.module('game').factory('TopicSettings', function () {
-  return window.topicSettings;
-});
```

Comment:
Removing all references to this, because `TopicSettings` is no longer a thing (used to contain the static word list definitions).